.github/workflows/gutenberg-packages-update.yml

```diff
@@ -0,0 +1,146 @@
+name: Gutenberg packages update
+
+on:
+  # Allow for the workflow to be manually run if needed.
+  workflow_dispatch:
+  schedule:
+    # Once a day (https://crontab.guru/once-a-day)
+    - cron: '0 0 * * *'
+
+jobs:
+  check-gutenberg-release:
+    name: Check for a new Gutenberg release
+    runs-on: ubuntu-latest
+    outputs:
+      latest-version: ${{ steps.latest-release.outputs.version }}
+      should-update: ${{ steps.release-status.outputs.outdated }}
+    steps:
+      - name: Get latest release version
+        id: latest-release
+        run: echo "::set-output name=version::$(gh api -X GET repos/wordpress/gutenberg/releases/latest --jq '.name')"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get release version from last PR
+        id: last-release
+        run: |
+          PR_TITLE=$(gh api -X GET search/issues -f q='${{ env.QUERY }}' -f sort='created' -f order='desc' --jq '.items.[].title' | head -n 1)
+          LAST_VERSION=$(node -e "'$PR_TITLE' && console.log( '$PR_TITLE'.match('v(.+) release')[1] )")
+          echo "::set-output name=version::$(echo "$LAST_VERSION")"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          QUERY: 'repo:ampproject/amp-wp is:pr is:open is:merged in:title Update Gutenberg packages after'
```

Comment:
Should this do an `author` check as well to be safe?

Reply:
Done in 46b6ef74746f5e44ea8b01a293e11f90141357a9.